CANReader/README.md

```diff
@@ -1,3 +1,20 @@
-This example demonstrates connecting to a CAN device and reading data.
+### CANReader
+Using the PolySync state machine this example does a simple CAN read in OK the state.
+This example will give you experience for the times you need a node to listen to a CAN channel.
 
-For more API examples and tutorials, visit the PolySync Knowledgebase [here](https://support.harbrick.com/hc/en-us/articles/216968458)
+### Hardware requirements
+- linuxcan-compatible hardware is connected, i.e. a Kvaser Leaf Light
+
+### Dependencies
+You need to have an CAN channel to run this example see:  [Connecting To A CAN Sensor](https://help.polysync.io/articles/configuration/runtime-node-configuration/connecting-to-a-can-radar-sensor/)
```

Comment:
This article is better suited than the tutorial, since its only focus is finding the CAN channel identifiers: https://help.polysync.io/articles/parse-can-hardware-and-circuit-identifiers/